packages/sirius-web/backend/sirius-web-papaya/src/main/java/org/eclipse/sirius/web/papaya/representations/table/PackageTableRepresentationDescriptionProvider.java

```diff
@@ -113,10 +120,16 @@ private List<Object> getAllTypesOfPackage(Package aPackage) {
         return result;
     }
 
-    private List<Object> getSemanticElements(VariableManager variableManager) {
-        return variableManager.get(VariableManager.SELF, PackageSpec.class)
+    private PaginatedData getSemanticElements(VariableManager variableManager) {
+        var optionalPagination = variableManager.get(TableRenderer.PAGINATION, CursorBasedPaginationData.class);
```

Comment:
The ADR was pretty clear, we need to have access to the various variables:

- cursor (which should be resolved as an object)
- direction
- size

etc

So please delete this concept of CursorBasedPaginationData, I need variables that can be manipulated by the end user using only the Sirius Web web interface. On top of that, one should not depend on `sirius-components-collaborative-xxx` to create a representation description.